lib/src/util/useEmptyComponent.ts

```diff
@@ -0,0 +1,35 @@
+import { useState, useEffect, useCallback, RefObject } from 'react';
+import useMutationObserver from '@rooks/use-mutation-observer';
+
+const mutationObserverConfig = { subtree: false, childList: true, attributes: false, characterData: false };
+
+/**
+ * Convenience hook that shows if a component is empty or not.
+ * This custom hook implements the useMutationObserver hook to check if there is a change in
+ * the parent component's children.
+ * @return a boolean if the component is empty.
```

Comment:
Document the parameters, especially since they're names aren't super obvious

Reply:
> Document the parameters, especially since they're names aren't super obvious

Added documentation for parameters.
29dc2caf958bd9983327ef1f982de59f97e9b475